CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ project( vecmem VERSION 1.8.0 LANGUAGES CXX )
 include( GNUInstallDirs )
 include( CTest )
 
+# Set the location of the installed CMake files.
+set( CMAKE_INSTALL_CMAKEDIR
+   "${CMAKE_INSTALL_LIBDIR}/cmake/vecmem-${PROJECT_VERSION}" )
```

Comment:
Is this conventional?

Reply:
This is the convention I went with for all of the R&D projects.

From my experience `lib/cmake` is the most used location for any project that doesn't need a `share/` directory otherwise. For projects that do use `share/`, the CMake files usually sit under `share/cmake/`. (Contrary to **all of that**, Gaudi and ATLAS use just `cmake/`. Go figure. :stuck_out_tongue:)

As for the final directory name, I like the idea of being able to have different versions of vecmem sitting side-by-side. That's why I don't just have `${CMAKE_INSTALL_LIBDIR}/cmake/vecmem` here. :thinking: (It's not something that we would necessarily ever do, but this at least allows for it.)

See https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure for all the possibilities that CMake understands. This here is one of the valid installation locations.